frontend/src/utils/broken-link-disabler.js

```diff
@@ -0,0 +1,18 @@
+window.$ = (id) => document.getElementById(id);
+const mixin = {
+  methods: {
+    isBrokenLink(link) {
```

Comment:
Seems like the links provided to this function will already be filtered and are of type String | undefined (undefined only if the link is broken). Can't we just check that the links are not undefined to determine if they're broken?

Reply:
Oh I see, thanks!